plugins/org.eclipse.n4js.generator.headless/src/org/eclipse/n4js/generator/headless/ConfiguredGeneratorFactory.java

```diff
@@ -65,7 +65,7 @@ ConfiguredGenerator getConfiguredGenerator(IN4JSProject project) {
 	 */
 	private void configureFSA(IN4JSProject project) {
 		File currentDirectory = new File(".");
-		File projectLocation = new File(project.getLocation().toFileString());
+		File projectLocation = new File(project._getLocation().toFileString());
```

Comment:
`_getLocation` is a temporary name for this method. Same as `getSafeLocation`.
The former should be removed whereas the `getSafeLocation` should be renamed to `getLocation`